README.md

```diff
@@ -37,34 +41,48 @@ Choose "Add Other" and browse to where you generated the file. You may need to c
 <img src="images/confirm.png" width="600" alt="Confirm">
 Confirm your selection and <strong>do not</strong> check the box to copy the file.
 
-> [!IMPORTANT]
-> The generated Linkmap.txt file must be included in your UI test target. You don't need to specificly use `"$(PROJECT_DIR)/Linkmap.txt"` as the path, only that whatever the `LD_MAP_FILE_PATH` is set is also the file that you include in Copy Bundle Resources.
+> [!NOTE]
+> You don't need to specifically use `"$(PROJECT_DIR)/Linkmap.txt"` as the path. Whatever value is set in `LD_MAP_FILE_PATH` is also the file that you include in Copy Bundle Resources.
 
-## Usage
+## Generating the order file 
 
-In a UI test, create an instance of `FaultOrderingTest` and optionally provide a closure to perform any necessary app setup. In most cases, this should include logging in to the app. Centering your UI test around a fully logged in session is strongly recommended, not only because it optimizes for the most common user experience, but also because it significantly improves the efficacy of this tool. By logging in, you allow much more of the app’s initial code to execute within the test context, allowing for a greater number of page fault reductions. The test case can then be executed.
+In a UI test case, create an instance of `FaultOrderingTest` and optionally provide a closure to perform any necessary app setup. 
+
+> [!TIP] In most cases, this should include logging in to the app. Centering your UI test around a fully logged in session is strongly recommended, not only because it optimizes for the most common user experience, but also because it significantly improves the efficacy of this tool. By logging in, you allow much more of the app’s initial code to execute within the test context, allowing for a greater number of page fault reductions.
 
 Example:
 
 ```swift
-let app = XCUIApplication()
-let test = FaultOrderingTest { app in
-  // Perform setup such as logging in
+import FaultOrderingTests
+import XCTest
+
+final class FaultOrderingLaunchTest: XCTestCase {
+    @MainActor
+    func testLaunch() throws {
+        let app = XCUIApplication()
+        let test = FaultOrderingTest { app in
+            // Perform setup such as logging in
+        }
+        test.testApp(testCase: self, app: app)
+    }
 }
-test.testApp(testCase: self, app: app)
 ```
 
+The test case can then be executed.
+
 > [!IMPORTANT]
-> This test should be run with a release build configuration, using the same compiler/linker optimizations that you would use on the App Store.
+> This test should run with the same compiler/linker optimizations that you would use for your App Store deliverable.
 
-### Accessing results
+### Device support
```

Comment:
I moved this section above "accessing results" since it's really a subdetail of how to run the ui test.